static/js/containers/HelpPage.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from "react";
+import { connect } from "react-redux";
+import R from "ramda";
+
+import WithDrawer from "./WithDrawer";
+import FAQ from "../components/FAQ";
+import { showFAQEntry } from '../actions/commonUi';
+
+class HelpPage extends React.Component {
+  toggleShowFAQ = R.curry((questionName, e) => {
+    const { dispatch } = this.props;
+
+    e.preventDefault();
+
+    dispatch(showFAQEntry(questionName));
```

Comment:
Nit: can we get rid of the blank line above? (I prefer to drop the blank line after `const` as well but I guess that's a semi-accepted pattern)